packages/webpack/template-webpack-plugin/src/LynxCacheEventsRuntimeModule.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import type { RuntimeModule } from 'webpack';
+
+import { RuntimeGlobals as LynxRuntimeGlobals } from '@lynx-js/webpack-runtime-globals';
+
+type LynxCacheEventsRuntimeModule = new() => RuntimeModule;
+
+export function createLynxCacheEventsRuntimeModule(
+  webpack: typeof import('webpack'),
+): LynxCacheEventsRuntimeModule {
+  return class LynxCacheEventsRuntimeModule extends webpack.RuntimeModule {
+    constructor() {
+      super('Lynx cache events', webpack.RuntimeModule.STAGE_TRIGGER);
```

Comment:
@colinaaa `webpack.RuntimeModule.STAGE_TRIGGER` ensures this runtime module is the last one in BTS:

<img width="917" height="773" alt="image" src="https://github.com/user-attachments/assets/c643caf5-5421-4856-a092-4acb4ba871d4" />